roles/cnf_cert/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Deprecation message
+  ansible.builtin.fail:  # noqa
+    msg: "[DEPRECATED] This module has been deprecated in favor of k8s_best_practices_certsuite and will be removed in a future release"
+  ignore_errors: true
```

Comment:
'failed_when: false' will be much clean although fail will provide red text

Reply:
done